backend/routes/user.ts

```diff
@@ -0,0 +1,28 @@
+import * as dotenv from 'dotenv';
+import path from 'path';
+dotenv.config({path: path.resolve(__dirname, "../.env")});
+import express from 'express';
+import cors from 'cors';
+import { PrismaClient } from '@prisma/client'
+import morgan from 'morgan';
+import passport from 'passport';
+import {Strategy as LocalStrategy} from 'passport-local';
+import crypto from 'crypto'
+import mysql from 'mysql2';
+import app from '../utils/app';
+import prisma from '../utils/db';
+
+const db = prisma; 
+
+app.post('/signup', 
+passport.authenticate('local', {failureRedirect: '/signup', failureMessage: true }),
+ function(req, res) {
+    console.log(req.body)
+ });
```

Comment:
I believe you're going to end up moving the route here right, just checking

Reply:
i will end up moving this